incubator/command-log/src/main/java/io/aklivity/zilla/runtime/command/log/internal/LoggableStream.java

```diff
@@ -387,6 +390,11 @@ private void onData(
             sequence - acknowledge + reserved, maximum, format("DATA [0x%016x] [%d] [%d] [%x] [0x%016x]",
                     budgetId, length, reserved, flags, authorization));
 
+        if (isPayloadIncluded)
+        {
+            onDataPayload(data);
```

Comment:
Inline the method body here, no need to separate.

Reply:
done